test/unit/tasks/ReportSpec.scala

```diff
@@ -48,19 +49,23 @@ class ReportSpec extends AnyWordSpec with Matchers with TryValues {
-          "https://www.qa.tax.service.gov.uk/accessibility-statement/test-service\ten\tTest (English)\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tfull\t0\t0\t1900-01-01\tfalse\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
-          "https://www.qa.tax.service.gov.uk/accessibility-statement/test-service\tcy\tTest (Welsh)\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tfull\t0\t0\t1900-01-01\tfalse\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
-          "https://www.qa.tax.service.gov.uk/accessibility-statement/english-service\ten\tEnglish Only\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tfull\t0\t0\t1900-01-01\tfalse\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
-          "https://www.qa.tax.service.gov.uk/accessibility-statement/with-milestones\ten\tWith Milestones\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tpartial\t2\t3\t2020-05-01\tfalse\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
-          "https://www.qa.tax.service.gov.uk/accessibility-statement/with-automated-testing\ten\tWith Automated Testing\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tpartial\t2\t3\t2020-05-01\ttrue\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
-          "https://www.qa.tax.service.gov.uk/accessibility-statement/draft-with-milestones\ten\tDraft With Milestones\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tpartial\t2\t1\t2020-05-01\tfalse\tdraft\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
-          "https://www.qa.tax.service.gov.uk/accessibility-statement/noncompliant\ten\tNoncompliant\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend	noncompliant\t0\t0\t1900-01-01\tfalse\tpublic\t1900-01-01\t2020-03-15\t2020-05-01\tHMRC\t\t\t\t",
-          "https://www.qa.tax.service.gov.uk/accessibility-statement/with-metadata\ten\tWith Metadata\thttps://www.tax.service.gov.uk/test/some.test.service\tsome.contact-frontend\tfull\t0\t0\t1900-01-01\tfalse\tpublic\t2020-02-28\t2020-03-15\t2020-05-01\tHMRC	Chief Digital & Information Officer (CDIO)\tDDC Worthing\tLive Services (Worthing)\tPublic beta"
+          "url\tlanguage\tserviceName\tserviceAbsoluteUrl\tcontactFrontendServiceId\tcomplianceStatus\tproblemCount\tmilestoneCount\tearliestMilestoneDate\tautomatedTestingOnly\tstatementVisibility\tserviceLastTestedDate\tstatementCreatedDate\tstatementLastUpdatedDate\tstatementType\tmonth\tyear\tBusiness Area\tDDC\tLive or Classic\ttype of Service\tIn Statement Service",
```

Comment:
@kyle-bowden Yup, this is from the DIAS specification linked in the ticket. All the naming of the columns has to be as they have specified so that it doesn't break their existing reporting tooling.